Devices/WioLTE/Device.h

```diff
@@ -0,0 +1,89 @@
+// Copyright GHI Electronics, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#define STM32F405xx 1
+
+#include <STM32F4.h>
+
+#define DEVICE_TARGET		STM32F4
+#define DEVICE_NAME			"WioLTE"
+#define DEVICE_MANUFACTURER	"Seeed"
```

Comment:
We should use the full name, Seeed Studio.

Reply:
'Seeed' is correct.
https://www.seeedstudio.com/about_us.html